src/common/helpers/config.js

```diff
@@ -16,19 +16,47 @@
  * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
  */
 
-import {readFileSync} from 'fs';
+import {existsSync, readFileSync} from 'fs';
+import {normalize} from 'path';
 
 
-/* Pull in environment-specific configuration. */
-const env = process.env.NODE_ENV || 'development'; // eslint-disable-line no-process-env,max-len
+let configContents;
 
-let configFileBasename = 'config';
-if (env === 'test') {
-	configFileBasename = 'test';
+// Use Default configuration (for Docker)
+function useDefaultConfig() {
+	/* Pull in environment-specific configuration. */
+	const env = process.env.NODE_ENV || 'development'; // eslint-disable-line no-process-env,max-len
+
+	let configFileBasename = 'config';
+	if (env === 'test') {
+		configFileBasename = 'test';
+	}
+	configContents =
+			readFileSync(`config/${configFileBasename}.json`);
 }
 
-const configContents =
-	readFileSync(`config/${configFileBasename}.json`);
+function checkConfigOverwrite() {
+	const args = process.argv;
+	const configIndex = args.indexOf('--config');
+
+	// Check for '--config' followed by 'configPathFile'
+	if (configIndex !== -1 && configIndex < args.length - 1) {
```

Comment:
I'm not sure I understand the reason for that  `configIndex < args.length - 1` check.
Could you explain that to me?

Reply:
Ah, I think I get it now. Is it to catch an empty `--config` flag, meaning `--config` _not_ followed by a path?

Reply:
yes